packages/itwinui-react/tsconfig.json

```
@@ -21,6 +21,7 @@
     "skipLibCheck": true,
     "isolatedModules": true,
     "verbatimModuleSyntax": true,
+    "types": ["vitest/globals", "@testing-library/jest-dom/vitest"],
```

nit: When I saved this file, it also reformated it. To keep the diff small, you can choose to not reformat it. But at least `types` would have been on multiple lines:

```suggestion
    "types": [
      "vitest/globals",
      "@testing-library/jest-dom/vitest"
    ],
```